docs/book/features/cns_telemetry.md

```diff
@@ -0,0 +1,42 @@
+<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable MD034 -->
+
+# vSphere CSI Driver - Cluster Distribution
+
+- [Introduction](#introduction)
+- [Deploy vSphere CSI driver with cluster-distribution](#vsphere-conf-cluster-distribution)
+- [Limitations](#limitations)
+
+**Note:** Feature to support `cluster-distribution` parameter in vSphere CSI driver is released with [v2.2.0](https://github.com/kubernetes-sigs/vsphere-csi-driver/releases/tag/v2.2.0).
+
+## Introduction <a id="introduction"></a>
+
+A new parameter, called `cluster-distribution`, is introduced to the `csi-vsphere.conf` file in an effort to collect telemetry data for analytics on vSphere Cloud Native Storage (CNS).
+
+This optional parameter needs to be set by the Kubernetes distribution.
+
+In future releases, this parameter will be made mandatory.
+
+## Deploy vSphere CSI driver with cluster-distribution<a id="vsphere-conf-cluster-distribution"></a>
+
+`cluster-distribution` field is added to the `Global` section of the `csi-vsphere.conf` file. An example `csi-vsphere.conf` file would look like:
+
+```cgo
+$ cat /etc/kubernetes/csi-vsphere.conf
+[Global]
+cluster-id = "<cluster-id>"
+cluster-distribution = "<cluster-distribution>" # optional
```

Comment:
Can we provide some examples of distributions, openshift, anthos, pks etc.

Reply:
done